README.md

```diff
@@ -34,13 +33,13 @@ Therefore, Node.js systems can serve more traffic because they are not idle or b
 
 When you develop application in Rails, you might use Haml to render server-side templates, then on the client-side you might use another framework and another template engine like Angular or Mustache. Those libraries are great, but consider that you have 100s of pages with a duplicate templates: one file for server-side rendering and another for browser rendering. To add complexity, this is an enterprise application which has been developed many years ago and you have 20+ people working on the project. How are you going to make changes to 200+ files? You need to make similar changes to two files and then change the tests as well. What if they are different languages? Imagine the nightmare. Sadly this is the case in most systems.
 
-Node is different because it has this concept called full stack, isomorphic or universal JavaScript. I prefer term full-stack JavaScript. No matter how you call it, the idea is that you can re-use files, libraries and modules between server and browser seamlessly. This is huge because we, as web developers amassed boatloads of great utilities and tools for browser JavaScript. Most of them are ready-to-go in Node environment with very minor modifications, e.g., using [Browserify](http://browserify.org). Same thing applies to books, blogs, screencasts. We know and improve JavaScript language (ES6, ES7, etc.) which put Node in a unique position. Honestly, I doubt Go or some other new platform can compete with Node in near future because of this unique combination which allows developers to have language to rule 'em all!
+Node is different because it has this concept called full stack, isomorphic or universal JavaScript. I prefer term full-stack JavaScript. No matter what you call it, the idea is that you can re-use files, libraries and modules between server and browser seamlessly. This is huge because we, as web developers, amassed boatloads of great utilities and tools for browser JavaScript. Most of them are ready-to-go in Node environment with very minor modifications, e.g., using [Browserify](http://browserify.org). Same thing applies to books, blogs, screencasts. JavaScript is continually improved, (ES6, ES7, etc.) which put Node in a unique position. Honestly, I doubt Go or some other new platform can compete with Node in near future because of this unique combination which allows developers to have language to rule 'em all!
```

Comment:
Reference to "I". Maybe say, "The term full-stack" is the most descriptive of the powers of Node" and again  replace second "I" in this paragraph with something like "There's no way Go or some other new platform can compete..."